docs/csv/example_files/ballot_style.txt

```diff
@@ -1 +1,3 @@
-id,image_uri,ordered_contest_ids,party_ids
\ No newline at end of file
+id,image_uri,ordered_contest_ids,party_ids
+bs00010,http://i.giphy.com/26BoCh3PgT8ai45ji.gif,oc2025,par02
```

Comment:
Could you provide an example in the docs here of a ballot style with multiple OrderedContest elements? Each ballot style here appears to have only one contest each.